### DISCUSSION TITLE
justify upgrade from 3.6.2?

### DISCUSSION THREAD

**Discussion by jamonpy** [jamulussoftware/jamulus#3399] [General]
Hello I am trying to justify upgrading my systems from version 3.6.2
I have > 10 raspberry pi's running jamulus 3.6.2 quite well for rehearsals.
Is there a particular qualitative improvement since 3.6.2 that I should test to justify the upgrade effort?
If I upgrade the server only - should the 3.6.2 clients  be compatible with server 3.11. ?
Thank you , I just trying to understand if the testing and logistics would be worth the effort.

**Comment by ann0see** [comment]
Yes absolutely. There are always bug fixes. For this jump there are also new features. JSON-RPC, a conductor "instrument". Clickable links (I assume),...

The more people are on newer versions, the better we can introduce new features. If you wait too long, things could break. 

Backwards compatibility for this version should shouldn't be a problem. You may need to reconfigure your settings in the clients. 

For your case the easiest way would be to use the repo: https://jamulus.io/wiki/Installation-for-Linux This will also allow auto upgrades in future.

**Comment by ann0see** [comment]
Also check out the change log: https://github.com/jamulussoftware/jamulus/blob/main/ChangeLog
